wave-app/src/model.py

```diff
@@ -16,6 +17,7 @@ async def model(q:Q):
 
     # Show model selection/load inputs.
     q.page['models'] = ui.form_card(box=ui.box('data'), items=[
+            ui.text(MODEL_INFO),
```

Comment:
same as `Data` tab -- please show the model "after" the model is loaded, i.e. after the "Load" button